src/main/java/seedu/address/model/person/Person.java

```diff
@@ -25,16 +26,19 @@ public class Person {
     private final Address address;
     private final Set<Tag> tags = new HashSet<>();
 
+    private final Set<Session> sessions;
+
     /**
      * Every field must be present and not null.
      */
-    public Person(Name name, Phone phone, Email email, Address address, Set<Tag> tags) {
-        requireAllNonNull(name, phone, email, address, tags);
+    public Person(Name name, Phone phone, Email email, Address address, Set<Tag> tags, Set<Session> sessions) {
+        requireAllNonNull(name, phone, email, address, tags, sessions);
         this.name = name;
         this.phone = phone;
         this.email = email;
         this.address = address;
         this.tags.addAll(tags);
+        this.sessions = new HashSet<>(sessions);
```

Comment:
this one might have to change to:
`
this.sessions.addAll(sessions);
`
Since its a Set<Session>, similar to Set<Tag>